pkg/model/openstackmodel/servergroup.go

```diff
@@ -152,6 +152,8 @@ func (b *ServerGroupModelBuilder) buildInstances(c *fi.ModelBuilderContext, sg *
 			Metadata:         igMeta,
 			SecurityGroups:   ig.Spec.AdditionalSecurityGroups,
 			AvailabilityZone: az,
+			ClusterName:      s(b.ClusterName()),
```

Comment:
Why aren't we using the new name creation in Line 106?

Reply:
old format uses that, but in new format the clustername needs to be passed as is to task (it is checked there)